### ISSUE TITLE
Admin theme issue with many menu items

### ISSUE THREAD

**Issue by ioweb-gr** [OpenMage/magento-lts#1515]
### Preconditions (*)
1. OpenMage 20.0.7

### Steps to reproduce (*)
1. Create many menu items 

### Expected result (*)
1. The menu items wrap around and are visible

### Actual result (*)
1. The menu items overlap with the search
![image](https://user-images.githubusercontent.com/20220341/112950165-a8c0aa80-9142-11eb-98c0-e6c096ce422a.png)

**Comment by ADDISON74** [comment]
This is more a feature request for a sandwich menu in OM theme than a bug. Personally I don't like developers who create links in the main menu for their products. An drop down link called Extensions is more than enough to group all extensions in one place. The other categories in Magento Backend have long been very well divided: Sales, Orders, Customers, Promotions, Newsletter, CMS, Reports, System.

**Comment by ioweb-gr** [comment]
I don't think it's a feature request because the menu item is overlapping with the search form. Wrapping or not the menu items, the menu elements should be clearly separated.

Whether it's cluttering or not is a different story though.

**Comment by ADDISON74** [comment]
Please download the latest Magento vanilla version 1945 from here: http://pubfiles.nexcess.net/magento/ce-packages/. Repeat your steps in order to reproduce this issue.

Frankly speaking that navigation menu wasn't planned to be a responsive one. Without a modification in templates and css files you would not be able to achieve what you expect. As I mentioned before it is a feature request not an OpenMage issue.

**Comment by ioweb-gr** [comment]
I kind of disagree on that one. The original vanilla instance of Magento 1.9.4.5 doesn't have this  issue because it wraps around no matter how many menu items you add, they're visible.
It's not a matter of responsiveness, it doesn't revert to a different layout or adjust to a hamburger menu or anything. Plain html is responsive by nature.

![image](https://user-images.githubusercontent.com/20220341/113039700-a3924880-91a0-11eb-9231-a74c80d5b159.png)

However with the admin theme introduced in OpenMage the search was moved in the same area as the menu causing this bug.

![image](https://user-images.githubusercontent.com/20220341/113039795-c15fad80-91a0-11eb-9569-d6ab5ef673e4.png)

**Comment by ADDISON74** [comment]
In OpenMage you can switch to Magento default theme for Backend. 

I am pretty sure the developer of the theme will see these posts and will reply.

**Comment by fballiano** [comment]
I agree with @ioweb-g

r, I think a new backend theme should be designed to be a bit more mobile-friendly (although probably nobody will use the backend from mobile... but also why not?).

Also, IMHO, if we don't have the work-power to manage a more-featured backend theme, we should remove app/design/adminhtml/default/openmage/template/page/header.phtml and adapt the CSS back and be happy :-)

**Comment by ioweb-gr** [comment]
No no don't get me wrong I just mentioned this as an issue that arose due to the new.theme. I like it but.it's good to fix some bugs if possible

**Comment by fballiano** [comment]
And I agree, the new theme is nice (although there are quite some accessibility issues due to color contrast) but, because we don't have a lot of work-force, let's try not to create small twiches to the stable core :-)

**Comment by aterjung** [comment]
Hi,

> Am 30.03.2021 um 22:34 schrieb ADDISON ***@***.***>:
> I am pretty sure the developer of the theme will see these posts and will reply.
> 
Sorry for answering late, i don’t have much time at them moment.

This is a known problem with the new admin theme. For my self, i don’t like these extensions putting there menu entries in the first level, but I don’t see what we could do to this. So I think it has to be fixed in the css to make the menu entries break like in the default theme. I hope i will find time to fix this in the next days.

**Comment by ioweb-gr** [comment]
I don't think this is something urgent :) Don't worry too much about it. I just reported it in case you could find some time to address it because it would help. Unfortunately the situation with the menus is not going to change in OpenMage

**Comment by sreichel** [comment]
> Unfortunately the situation with the menus is not going to change in OpenMage

Isn't it possible to re-order the menu entries via own xml? (never tried)

**Comment by ioweb-gr** [comment]
@sreichel We have a customer with 35 Amasty modules installed and about 20 others from other companies. There's no rescue in that haha. I haven't actually tried reordering via a local XML to put them under a single menu to be honest. I don't know if it's possible

**Comment by Flyingmana** [comment]
So I had a short look into the Issue. (its clear we should fix it in OpenMage, as we introduced in with the new theme)

In my Test instance (19.4.10 and 20.0.4) the Search header has a width of  347px, and the menue is already breaking into next line.

![image](https://user-images.githubusercontent.com/237319/113472256-475e4b80-9462-11eb-9cc2-1ba3029bcb86.png)

The menue already has a padding because of the logo on the left I guess, thats 70px. But we need more on the right side, so its matching the search header

```
.nav-bar {
    background: #202856;
    padding: 0 70px;
}
```

@ioweb-gr is this correct for your case, too?
Are the items breaking into next line if you make the window a bit smaller?
The li elements have a float:left, so they should break at one point.

**Comment by ioweb-gr** [comment]
@Flyingmana Yes it is like this, for me as well however in your case you're having a visible menu because of the demo store notice. If you remove that the menu items will still go behind the search :)

I haven't had a chance to test your css but I'll do so asap

**Comment by elidrissidev** [comment]
I think this issue is already fixed in v19.4.14, so this issue can be closed. It also seems to be closely related to #1214.

**Comment by ADDISON74** [comment]
@ioweb-gr - Could you please check #1214 and confirm if it solves your issue?

**Comment by seifer7** [comment]
Just chiming in to say that this issue is fixed in the current stable release (20.0.20.)

**Full length menu**
![image](https://github.com/OpenMage/magento-lts/assets/13601073/f5fbbd6e-84e4-4f03-a4f4-929390b060a8)

**Wrapped menu**
![image](https://github.com/OpenMage/magento-lts/assets/13601073/dd96a491-5921-4b29-aae7-78a5c52d0c35)
